src/olympia/addons/indexers.py

```diff
@@ -23,6 +23,8 @@ def get_mapping(cls):
             'properties': {
                 'max': {'type': 'long'},
                 'min': {'type': 'long'},
+                'max_human': {'type': 'string', 'index': 'no'},
+                'min_human': {'type': 'string', 'index': 'no'},
```

Comment:
I could have done without that and converted back the version integer to a string in the serializer, but it turns out that the code we have to convert a version integer back to a string is not 100% fool-proof, it has issues with trailing zeros and `*`, so I preferred just storing the actual human-readable version.